.github/workflows/go.yml

```diff
@@ -14,7 +14,7 @@ on:
 
 concurrency:
   group: "go-${{ github.workflow }}-${{ github.event.number || github.ref }}"
-  cancel-in-progress: "${{ github.event.action == 'push' || github.event.action == 'pull_request' }}"
+  cancel-in-progress: "${{ github.event_name == 'pull_request' }}"
```

Comment:
@joshuasing I've just thought of this: do we also want to cancel on `push` to the same branch and workflow as well? 🤔

Reply:
@ClaytonNorthey92 I think not cancelling on push is okay, and could cause issues when invoked by `workflow_dispatch`.